src/plots/polar/polar.js

```diff
@@ -129,6 +128,8 @@ proto.updateLayers = function(fullLayout, polarLayout) {
             switch(d) {
                 case 'frontplot':
                     sel.append('g').classed('scatterlayer', true);
+                    // TODO add option to place in 'backplot' layer??
+                    sel.append('g').classed('barlayer', true);
```

Comment:
At the moment, bars are drawn above the grid lines.

But perhaps, we could add an option to draw bars below grid lines?

Reply:
> But perhaps, we could add an option to draw bars below grid lines?

True, for polar bars that could be nice to help with relative sizes. Not necessary in v1 though.